apps/s_cb.c

```diff
@@ -1373,27 +1428,26 @@ static int security_callback_debug(const SSL *s, const SSL_CTX *ctx,
         {
             const unsigned char *salg = other;
             const char *sname = NULL;
-            switch (salg[1]) {
-            case TLSEXT_signature_anonymous:
-                sname = "anonymous";
-                break;
-            case TLSEXT_signature_rsa:
-                sname = "RSA";
-                break;
-            case TLSEXT_signature_dsa:
-                sname = "DSA";
-                break;
-            case TLSEXT_signature_ecdsa:
-                sname = "ECDSA";
-                break;
-            }
+            int raw_sig_code= (salg[0]<<8) + salg[1]; /* always big endian (msb, lsb) */
+                /* raw_sig_code: signature_scheme from tls1.3, or signature_and_hash from tls1.2 */
 
-            BIO_puts(sdb->out, OBJ_nid2sn(nid));
-            if (sname)
-                BIO_printf(sdb->out, ", algorithm=%s", sname);
-            else
-                BIO_printf(sdb->out, ", algid=%d", salg[1]);
-            break;
+            BIO_printf(sdb->out, "%s", nm);
+
+            sname= lookup(raw_sig_code, signature_tls13_scheme_list, NULL);
+            if (sname) {
```

Comment:
Style nit: Make this `sname != NULL`

Reply:
fixed this (at-least in my fork), and the other coding style note. I'll try to send a printed/signed/scanned cla within 24 hours.